src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -173,8 +173,9 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
                         config.AddCommandLine(args);
                     }
                 })
-                .ConfigureLogging(logging =>
+                .ConfigureLogging((hostingContext, logging) =>
                 {
+                    logging.UseConfiguration(hostingContext.Configuration.GetSection("Logging"));
```

Comment:
There aren't any tests in MetaPackages yet :'(
See https://github.com/aspnet/MetaPackages/pull/46